contrib/add_license_to_files.jl

```diff
@@ -71,8 +71,9 @@ const old_license = ""
 ### END CONFIG HERE
 
 
-function check_lines!(path::AbstractString, lines::Vector, checktxt::AbstractString,
-                                                 prefix::ASCIIString, oldcheck::Bool)
+function check_lines!(
+    path::AbstractString, lines::Vector, checktxt::AbstractString,
+    prefix::AbstractString, oldcheck::Bool)
```

Comment:
how about one extra indent on the continuation line? otherwise the leading `(` and the rest of the arguments being exactly inline with the function body looks a bit off